ibm/mas_devops/roles/mirror_images/templates/auth-secret.json.j2

```diff
@@ -1,9 +1,19 @@
 {
   "auths": {
-{% if ibm_auth is defined and ibm_auth != ":" %}
-    "cp.icr.io": {
+{% if artifactory_auth is defined and artifactory_auth != ":" %}
+    "docker-na-public.artifactory.swg-devops.com": {
+      "email":"{{ artifactory_username }}",
+      "auth":"{{ artifactory_auth | b64encode }}"
+    },
+    "wiotp-docker-local.artifactory.swg-devops.com": {
```

Comment:
We removed wiotp-docker-local.artifactory.swg-devops.com from the image pull secret in the other file, so we may as well remove it from here too for consistency.  This old domain won't work now anyway as it's been shutdown.